.ionblaster.json

```diff
@@ -1,7 +0,0 @@
-[
```

Comment:
for the sake of minimizing merge conflicts when rebasing, maybe it's better to not remove any files.